src/ir/ir_visitor.h

```diff
@@ -31,7 +31,7 @@ class IRVisitor {
  public:
   virtual ~IRVisitor() {}
   virtual Result Visit(CopyConst<IsConst, Module>& module) = 0;
-  virtual Result Visit(CopyConst<IsConst, Block>& operation) = 0;
+  virtual Result Visit(CopyConst<IsConst, Block>& block) = 0;
```

Comment:
FYI This looked like a typo so I deliberately fixed this here.

Reply:
Thanks! I wish cpp kept us honest about these.